js/logos.json

```diff
@@ -438,4 +438,16 @@
 	"title": "Whogohost",
 	"filename": "whogohost",
 	"category": ["Web Host", "Domain Registrar"]
+}, {
+	"title": "TechCabal",
+	"filename": "techcabal",
+	"category": ["Blog"]
```

Comment:
```suggestion
	"category": ["Media"]
```

Hey, I think this a more appropriate categorization for TC.

Reply:
@fathermerry I agree. Fixed, thanks.